src/validations/class.js

```diff
@@ -12,4 +12,49 @@ const validateId = (req, res, next) => {
   });
 };
 
-module.exports = { validateId };
+const validateCreateClass = (req, res, next) => {
+  const classValidation = Joi.object({
+    hour: Joi.string()
+      .pattern(/^([01]?[0-9]|2[0-3]):([0-5][0-9])$/)
+      .required()
+      .messages({
+        'string.pattern.base': 'Hour format is HH:mm',
+      }),
+
+    day: Joi.date()
```

Comment:
Day has to be an enum (monday, thuesday, etc)
Changes:
- Replace date() with string()
- Add the valid options (all days of the week)
- Change the message